### PR TITLE
Request reviewers on created PR

### PR DESCRIPTION
**Pull request by jgsogo**

### CODE REVIEW

File: .ci/on_conan_release.jenkinsfile [hunk 1]
```diff
@@ -64,26 +64,44 @@ node('Linux') {
...
+                requestBody: jsonContent,
+            )
+
+            Map<String, Object> responseData = readJSON(text: response.content)
+            prNumber = responseData.id as int
+        }
+
+        stage('Request reviewers') {
+            Map data = [:]
+            data.reviewers = ['uilianries', 'jgsogo']
```
Comment (uilianries) [inline .ci/on_conan_release.jenkinsfile:96]: @SSE4  do you want to be a requested reviewer? You usually read CDT PRs as far as I see.

Comment (SSE4) [reply .ci/on_conan_release.jenkinsfile:96]: I can try to do my best, if it helps moving things forward. as I can see there is not so much activity compare to CCI, so it shouldn't require that much attention.

Comment (jgsogo) [reply .ci/on_conan_release.jenkinsfile:96]: This affects only to this kind of PR created by the bot: https://github.com/conan-io/conan-docker-tools/pull/423, one every month.

Comment (jgsogo) [reply .ci/on_conan_release.jenkinsfile:96]: You can always add/remove you later, in any case.

@uilianries , please review (this one) and merge, we need to try it using CI

**Review by uilianries — APPROVED** [review]

**Pull request merged**